definitions.js

```diff
@@ -23,7 +23,10 @@ export const UNITS = [
 ];
 
 export const UNIT_CURRENCIES = {
+  ALL: 'ALL',
```

Comment:
Adding currencies here probably also means the latest exchange rate should be available. They must thus be added.

Reply:
How do you deal with the situation where it is not the case (e.g. a currency disappears in favor of the euro)?

Reply:
Then we should make sure an error is caused during conversion.

Reply:
I've found the exchange rates, but this could be a problem in the future.